plugins/codeclimate/src/api/mock/mock-api.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CodeclimateData } from '../codeclimate-data';
+import { CodeclimateApi } from '../codeclimate-api';
+import moment from 'moment';
```

Comment:
Apols, another quick one since I noticed this.

The project has a published ADR regarding standardizing on the Luxon data time library:
https://backstage.io/docs/architecture-decisions/adrs-adr010

Ideally, could this be refactored to use that instead for consistency across the project as a whole? You'll find numerous examples in the repo to borrow from if it helps?

Reply:
Also: https://backstage.io/docs/architecture-decisions/adrs-adr012

Reply:
Done, changed `moment` for `luxon` and `humanize-duration`